### DISCUSSION TITLE
Website images do not appear

### DISCUSSION THREAD

**Discussion by insideinside** [leokhoa/laragon#382] [General]
Hi.

I had to install Windows again today. The Laragon folder stayed in the same place. I installed Laragon again too and it worked again but when I access the sites locally the images do not appear.

When I try to access the file via browser like this:

http://localhost:90/mysite/wp-content/uploads/2022/08/logo.png

The message appears: You don't have permission to access this resource.

It seems to be a folder permission issue.

Does anybody know how to solve this? I use Windows 10.

Thank you for your help.

**Comment by Pen-y-Fan** [comment]
See my blog: [How to set up Laragon on a new Windows computer](https://pen-y-fan.github.io/2022/11/07/how-to-set-up-laragon-on-a-new-windows-computer/), part 2 has details on permissions.

I would expect your wp site to be accessed via [pretty URLs](https://laragon.org/docs/pretty-urls.html). http://mysite.test

**Comment by peakyharmer567** [comment]
Hi.
I am also facing similar problem like this on my website [engraved wood](https://etchlaser.mystrikingly.com/).
(The message appears: You don't have permission to access this resource).
If you get solution of this mention me here.
Thanks.

**Comment by insideinside** [comment]
I managed to solve the problem was permission on the same folder.

**Comment by harmwalex234** [comment]
Hi.
My website [netflix free watch](https://apklookup.com/kinemaster-pro-mod-apk-free-download/) is also experiencing similar issues.
(The message appears: This resource is not accessible to you.)
Please let me know if you find a solution to this problem.
Thanks.

**Comment by williamsonparker289** [comment]
Hey there!
The same problem is occurring on my [Trading With Pivot Points](https://tradingstrategyguides.com/how-to-trade-with-pivot-points/) website.
You do not have permission to access this resource (the message appears).
Please let me know if you find a solution to this.
Thank you.
